[blog] New posts (#22607)
mui-org/material-ui: docs/pages/blog/spotlight-damien-tassone.md

---
title: A spotlight on Damien Tassone joining the team

> description: Damien Tassone has joined our company. He's the first full-time member to focus on the enterprise components of Material-UI.

Reviewer: ```suggestion
description: Damien Tassone has joined our Material-UI. He's the first full-time member to focus on enterprise components.
```

date: 2020-09-15T00:00:00.000Z
authors: ['oliviertassinari']
---

> A few months ago, right in the middle of the COVID-19 outbreak, [Damien Tassone](https://twitter.com/madKakoO) has joined our company. He's the first full-time member to focus on the enterprise components of Material-UI. Back then, we had only made a quick mention of it. It's never too late to properly introduce him.

Reviewer: ```suggestion
A few months ago, right in the middle of the COVID-19 outbreak, [Damien Tassone](https://twitter.com/madKakoO) joined Material-UI. He's the first full-time member to focus on enterprise components. Back then, we only made a quick mention of it. It's never too late to introduce him properly.
```

> Previously to joining, Damien has been working in the finance industry for a decade in London. Part of the initial, Agile Market team at RBS, and more recently leading a large Commodity Trading application at Adaptive, Damien has developed his React experience as part of large projects where Components driven design plays a huge role in delivering application successfully. He loves TypeScript.

Reviewer: ```suggestion
Prior to joining Material-UI, Damien worked in the finance industry in London for a decade. Part of the initial Agile Market team at RBS, and more recently leading a large Commodity Trading application at Adaptive, Damien has developed his React experience as part of large projects where Components driven design plays a huge role in successfully delivering applications. He loves TypeScript.
```

> Previously to joining, Damien has been working in the finance industry for a decade in London. for a decade in London. Part of the initial, Agile Market team at RBS, and more recently leading a large Commodity Trading application at Adaptive, Damien has developed his React experience as part of large projects where Components driven design plays a huge role in delivering application successfully. He loves TypeScript.

Reviewer 2: rm `. for a decade in London` written twice

> I was impressed with his energy and passion for solving UI problems.

Reviewer: ```suggestion
```

> Right from the beginning, he has made fast and significative progress. He has been an amazing addition to the team, thanks to him, we are on the verge to release a new advanced component: the [Data Grid](https://material-ui.com/components/data-grid).

Reviewer: ```suggestion
Right from the start, Damien has made fast and significant progress. He has been an amazing addition to the team, and thanks to him, we are on the verge of releasing a new advanced component: the [Data Grid](https://material-ui.com/components/data-grid).
```

<img src="/static/blog/spotlight-damien-tassone/data-grid.png" style="margin-bottom: 3rem;" alt="Data Grid" />

> You can follow Damien on [Twitter](https://twitter.com/madKakoO) to see his progress. The enterprise team will continue developing the data grid component, and progressively extend its scope of interest to new advanced components, as we grow the team. A quick view at [the enterprise roadmap](https://github.com/mui-org/material-ui-x/projects/1).

Reviewer: ```suggestion
The enterprise team will continue developing the data grid component, and progressively extend to new advanced components as we grow the team. Here's a quick view of [the enterprise roadmap](https://github.com/mui-org/material-ui-x/projects/1).

You can follow Damien on [Twitter](https://twitter.com/madKakoO).
```
